docs/dataset_curation_guidelines.md

```diff
@@ -0,0 +1,141 @@
+# Dataset Curation Guidelines for Loading and File Placement
+
+This guide outlines the process for curating datasets and specifies where to place the curation 
```

Comment:
Suggest amend to "and specifies the location where curation files should be placed for proper loading"